src/cmd.ts

```diff
@@ -134,14 +134,20 @@ function createCommandWithSharedOptions(name, description) {
 		)
 		.option(
 			"-e, --include-episodes",
-			"Include single-episode torrents in the search",
+			"Include all episode torrents in the search",
```

Comment:
```suggestion
			"Include all episode torrents in the search (including from season packs)",
```

Reply:
Will commit the removal of this redundant option.